packages/deployments/contracts/contracts/lib/Connext/ConnextUtils.sol

```diff
@@ -49,13 +49,38 @@ library ConnextUtils {
     return keccak256(abi.encode(transfer));
   }
 
+  function verifyRouterPermit(
+    bytes32 _permit,
+    address[] calldata _routers,
+    bytes[] calldata _signatures
+  ) external view {
+    for (uint i = 0; i < _signatures.length; i++) {
+      // TODO: If the sender *is* the router, skip
+      if (msg.sender == _routers[i]) {
+        continue;
+      }
```

Comment:
need to check this against the relayer checks that happen in execute as well (though really the router shouldn't be sending this in the majority of cases

Reply:
wym? This is contingent on there only being one signature/verification method, i.e. the relayer signature isn't checked.